src/applications/vaos/appointment-list/components/cards/pending/AppointmentRequestListItem.jsx

```diff
@@ -84,32 +79,30 @@ export default function AppointmentRequestListItem({
               appointment={appointment}
             />
           )}
-          {isExpressCare && facility?.name}
-          {!isCC &&
-            !isExpressCare && (
-              <VAFacilityLocation
-                facility={facility}
-                facilityName={facility?.name}
-                facilityId={facilityId}
-              />
-            )}
+          {facility?.name}
```

Comment:
I think this should be removed, since it was only shown when `isExpressCare` was true.

Reply:
Are you referring to line 82?

Reply:
Yeah, the `{facility?.name}` line

Reply:
@LanceGundersen This is still outstanding, I think